fast_pred/src/fastlightfm.h

```diff
@@ -22,23 +22,26 @@ class FastLightFM {
   cnpy::NpyArray user_biases;
 
   int no_components;
+  int number_of_users;
+  int number_of_items;
 
   double item_scale;
   double user_scale;
 
   struct FastLightFMCache *lightfm_cache;
 
+  void init();
+
 public:
   FastLightFM()
-      : item_features(nullptr), user_features(nullptr), no_components(0),
```

Comment:
Maybe we should keep this initialisation?

Reply:
yes, all the member variables needs to be initialized in the constructor, otherwise they are in random value in C++  :)